src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: AirGap Networks
+description: Installing Okteto in AirGap Networks
+sidebar_label: AirGap Networks
+id: airgap
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# AirGap Networks<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works on AirGap Networks, but it requires custom configuration.
+This guide has been crafted to help you overcome these hurdles, but we encourage you to [talk to us](https://www.okteto.com/get-demo/) to guide you during the installation.
+
+
+## Step 1: Disable Okteto Telemetry
+
+By default, Okteto sends [telemetry data](self-hosted/helm-configuration.mdx#telemetry) to help us understand usage patterns and prioritize improvements
+In air-gapped environments, you can disable telemetry by adding the following to your Okteto Helm configuration:
+
+```yaml
+telemetry:
+  enabled: false
+```
+
+Disabling telemetry will automatically stop Okteto CLI analytics for all developers.
+
+## Step 2: Host Images in your Private Registry
+
+By default, all the images needed to deploy the Okteto Platform, and the images used by the Okteto CLI, are hosted in DockerHub.
+In air-gapped environments, you’ll need to host these images in your private registry.
+
+### Push Images to your Private Registry
+
+For simplicity, we recommend using the same repository names and tags in your private registry.
```

Comment:
I think it's better to remove this line so we don't give people ideas that there are other ways. It's better if we recommend a single way of doing things (in this case, to upload to the private registry with the same name/tag)